python-package/lightgbm/dask.py

```diff
@@ -176,12 +177,74 @@ def _train_part(
     else:
         group = None
 
+    # construct local eval_set data.
+    local_eval_set, local_eval_sample_weight, local_eval_group = None, None, None
```

Comment:
```suggestion
    local_eval_set = None
    local_eval_sample_weight = None
    local_eval_group = None
```

can you please move these to individual lines? I think that's a bit easier to read, but more importantly it makes [the git blame output](https://github.com/microsoft/LightGBM/blame/fd094d2719a04b984b376a665b6c3e5e1aff2967/python-package/lightgbm/dask.py) more useful